app/cdap/components/WranglerGrid/SelectColumnPanel/styles.ts

```diff
@@ -0,0 +1,240 @@
+} from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import CheckBoxOutlineBlankOutlinedIcon from '@material-ui/icons/CheckBoxOutlineBlankOutlined';
+import CheckBoxOutlinedIcon from '@material-ui/icons/CheckBoxOutlined';
+import CloseIcon from '@material-ui/icons/Close';
+import SearchIcon from '@material-ui/icons/Search';
+import styled, { css } from 'styled-components';
+import IndeterminateCheckBoxOutlinedIcon from '@material-ui/icons/IndeterminateCheckBoxOutlined';
+
+const cellFontCSS = css`
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  letter-spacing: 0.15px;
+`;
+
+export const ColumnInnerWrapper = styled(Box)`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const ColumnWrapper = styled(Box)`
+  height: 90%;
+`;
+
+export const FlexWrapper = styled(Box)`
+  align-items: center;
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+`;
+
+export const PanelContainer = styled(Container)`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 0;
+`;
+
+export const SearchIconButton = styled(IconButton)`
+  padding: 5px 0px 5px 5px;
+
+  &.MuiIconButton-root:hover {
+    background-color: transparent;
+  }
+  & .MuiTouchRipple-root {
+    display: none;
+  }
+`;
+
+export const SearchInputField = styled.input`
+  margin-right: 5px;
+  border: none;
+  border-bottom: ${(props) =>
+    props.isFocused ? `1px solid ${grey[700]}` : '1px solid transparent'};
+  margin-bottom: 5px;
+  outline: none;
+`;
+
+export const SearchWrapper = styled(Box)`
+  position: relative;
+  display: flex;
+`;
+
+export const StyledButton = styled(Button)`
+  width: 73px;
+  height: 36px;
+  background: #3367d6;
+  box-shadow: 0px 2px 4px rgba(70, 129, 244, 0.15);
+  border-radius: 4px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 36px;
+  letter-spacing: 1.25px;
+  color: #ffffff;
+  margin-top: 30px;
+  text-transform: none;
+  margin-bottom: 20px;
+  margin-right: 8px;
+  &:hover {
+    background: #3367d6;
+  }
+  &.Mui-disabled {
+    background: ${grey[300]};
+  }
+`;
+
+export const StyledCheckbox = styled(Checkbox)`
+  padding: 0;
+  vertical-align: text-top;
+  & .MuiIconButton-label {
+    width: 24px;
+    padding-right: 12px;
+  }
+`;
+
+const iconSize = css`
+  width: 24px;
+  height: 24px;
+`;
+
+export const CheckboxCheckedIcon = styled(CheckBoxOutlinedIcon)`
+  &.MuiSvgIcon-root {
+    ${iconSize}
+  }
+`;
+export const CheckboxBlankIcon = styled(CheckBoxOutlineBlankOutlinedIcon)`
+  &.MuiSvgIcon-root {
+    ${iconSize}
+  }
+`;
+
+export const CheckboxIndeterminateIcon = styled(IndeterminateCheckBoxOutlinedIcon)`
+  &.MuiSvgIcon-root {
+    ${iconSize}
+  }
+`;
+
+export const StyledClearIcon = styled(CloseIcon)`
+  &.MuiSvgIcon-root {
+    ${iconSize}
+  }
+`;
+
+export const StyledRadio = styled(Radio)`
+  &.MuiRadio-colorPrimary.Mui-checked {
+    color: #3367d6 !important;
```

Comment:
I see a lot of importants in this file - if you _have_ to use them, which i doubt since you can use higher specificity, leave a comment mentioning why